friskby/views/quick.py

```diff
@@ -66,9 +87,13 @@ def get(self , request):
                 'pm10': data10list[0]["value"],
                 'pm25list': data25list, 
                 'pm10list': data10list,
-                'time': time}
+                'time': time_pp,
+                'isotime': time}
             device_rows.append(row)
 
+        algo_end = dt.now()
+        algo_delta = algo_end - algo_start
+        print('total time used: %.2f sec' % algo_delta.total_seconds())
```

Comment:
Actually, I wanted to keep it if you don't mind.  I've been writing these lines of codes now every time I do something in this part just to ensure that I don't make changes that make Quick (much) slower.